main.tf

```diff
@@ -23,7 +27,14 @@ resource "google_container_cluster" "primary" {
 
   maintenance_policy {
     daily_maintenance_window {
-      start_time = "04:00"
+      # 9am EST
+      # For maintenance windows in general,
+      # people usually choose a time of least-use.
+      # The nature of Airflow is such that the jobs
+      # are likely to run in those same windows, so
+      # it's best to just choose a time where support
+      # will likely be available.
+      start_time = "13:00"
```

Comment:
How about making this a TF variable as this might vary from company to company.

Maybe  default the TF var with "13:00" and let the users override it if it is not the same for them.